sections/csgcn/csgcn-chapter.tex

```diff
@@ -299,7 +299,7 @@ \subsection{Model intuition}\label{subsec:csgcn_adj_intuition}
 The adjacency matrix will now include interactions, what context the users have interacted in, and the context in which items have been interacted with.
 This approach is unable to capture which specific contexts the different interactions occurred in.
 It only captures whether or not the user or item has had an interaction in the given context.
-This differs from \textit{CSGCN-IS} since we now model context in a similar fashion to side-information, where each context value has a single vector representation instead of having a vector representation for each context value and item pair.
+This differs from \textit{CSGCN-IS} since we now model context in a similar fashion to side-information, where each context value has a single embedding instead of having a embedding for each context value and item pair.
```

Comment:
```suggestion
This differs from \textit{CSGCN-IS} since we now model context in a similar fashion to side-information, where each context value has a single embedding instead of having an embedding for each context value and item pair.
```